api/controllers/site.js

```diff
@@ -104,7 +107,7 @@ module.exports = {
     const userId = Number(req.params.user_id);
     let site;
 
-    if (isNaN(siteId) || isNaN(userId)) {
+    if (_.isNaN(siteId) || _.isNaN(userId)) {
```

Comment:
s this necessary? this style isNaN is not used throughout the codebase

Reply:
The isNaN was throwing linting error's so I used the underscore version.